catalog/ngis-cat/projects_ngis-cat_assets_DEA_NIDEM.jsonnet

```diff
@@ -0,0 +1,179 @@
+local id = 'projects/ngis-cat/assets/DEA/NIDEM';
+local subdir = 'ngis-cat';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'National Intertidal Digital Elevation Model 25m 1.0.0',
+  'gee:type': ee_const.gee_type.image,
+  
+  description: |||
+    The National Intertidal Digital Elevation Model (NIDEM; Bishop-Taylor et al. 2018, 2019) is a continental-scale elevation dataset for Australia's exposed intertidal zone. 
+    NIDEM provides the first three-dimensional representation of Australia's intertidal sandy beaches and shores, tidal flats and rocky shores and reefs at 25 m spatial resolution, 
+    addressing a key gap between the availability of sub-tidal bathymetry and terrestrial elevation data. NIDEM was generated by combining global tidal modelling 
+    with a 30-year time series archive of spatially and spectrally calibrated Landsat satellite data managed within the Digital Earth Australia (DEA) platform. 
+    NIDEM complements existing intertidal extent products, and provides data to support a new suite of use cases that require a more detailed understanding 
+    of the three-dimensional topography of the intertidal zone, such as hydrodynamic modelling, coastal risk management and ecological habitat mapping.
+    The product: https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics

```

Comment:
Links must be in `[]()` to be active in any markdown.  Here and below

Reply:
Done